{
    "type": "bundle",
    "id": "bundle--0ff9b9b6-c5c3-4d1d-a199-a8801db7d334",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "id": "relationship--de071e4d-eee7-4ad5-96b6-fe83bfe3e25a",
            "type": "relationship",
            "created": "2020-12-30T16:39:34.375Z",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "external_references": [
                {
                    "source_name": "Cyble Egregor Oct 2020",
                    "url": "https://cybleinc.com/2020/10/31/egregor-ransomware-a-deep-dive-into-its-activities-and-techniques/",
                    "description": "Cybleinc. (2020, October 31). Egregor Ransomware \u2013 A Deep Dive Into Its Activities and Techniques. Retrieved December 29, 2020."
                }
            ],
            "modified": "2020-12-30T16:39:34.375Z",
            "description": "[Egregor](https://attack.mitre.org/software/S0554) has checked for the LogMein event log in an attempt to encrypt files in remote machines.(Citation: Cyble Egregor Oct 2020)",
            "relationship_type": "uses",
            "source_ref": "malware--cc4c1287-9c86-4447-810c-744f3880ec37",
            "target_ref": "attack-pattern--4061e78c-1284-44b4-9116-73e4ac3912f7",
            "x_mitre_version": "1.0",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}